test/test.sh

```diff
@@ -759,7 +768,14 @@ test -e ../afl-qemu-trace && {
       test "$SYS" = "i686" -o "$SYS" = "x86_64" -o "$SYS" = "amd64" -o "$SYS" = "i86pc" -o "$SYS" = "aarch64" -o ! "${SYS%%arm*}" && {
         $ECHO "$GREY[*] running afl-fuzz for persistent qemu_mode, this will take approx 10 seconds"
         {
-          export AFL_QEMU_PERSISTENT_ADDR=`expr 0x4$(nm test-instr | grep "T main" | awk '{print $1}' | sed 's/^.......//' )`
+          if file test-instr | grep -q "32-bit"; then
```

Comment:
For usability, a tool/shellscript to map a relocatable addr to the qemu-internal load addr might be beneficial. Using `AFL_QEMU_PERSISTENT_ADDR` needs a lot of internal knowledge about qemu at the moment.

Reply:
I thought about this too. but how? it only applies if PIE, AFL_QEMU_PERSISTENT_ADDR is set but not set to a 0x4.... address.  IMHO for now the documenation in the README and the FATAL message in afl-fuzz are ok. but because I dont know of a better way :)

Reply:
We could finds out using readelf (or some other way) if it's PIE, then
no_pie => return input
x86 => add 0x4...
x64 => ...

Reply:
lets put that on the todo list